### ISSUE TITLE
All fields in Daily Rounds table to treat no value specified as `undefined` instead instead of colaescing it with 0 or -1 in some fields

### ISSUE THREAD

**Issue by nihal467** [coronasafe/care_fe#6985]
**Describe the bug**

all the nutrition section field's in the patient critical care is saving the values as 0, when we save critical care without 

**To Reproduce**

1. Go to patient critical care and open nutrition section
2. save the critical care without filling any nutrition section fields and save it
3. go to the nutrition tab in the patient consultation page

**Expected behavior**
The empty field in the nutrition section should be a empty object not a 0

## Tasks

This can be done as a two-part PR, front-end followed by back-end.

- [x] #7070
- [x] https://github.com/coronasafe/care/issues/1810

**Screenshots**

<img width="959" alt="image" src="https://github.com/coronasafe/care_fe/assets/57055998/a6bd39e0-2ec8-468e-8ea5-91dedfd2e106">

**Comment by rithviknishad** [comment]
As discussed, 

For ALL THE FIELDS IN DAILY ROUNDS TABLE, we need to send `null` for if there's no value in the input field. We should not internally coalesce with 0.

This can be done as a two-step exercise:

- Update FE to have a clear distinction between null values and non-null values.
- Custom Migration to clean existing records.

**Comment by AshrafMd-1** [comment]
@rithviknishad Except nutrition all others show `-` if there is no data 
and in io balance 
it's due to the fact that the intake and out-take starts from 0
![image](https://github.com/coronasafe/care_fe/assets/98876115/6dcc293f-f4d8-437c-be9b-799d7d32a4f3)
as you can see now there is no below values 
![image](https://github.com/coronasafe/care_fe/assets/98876115/3daafc61-651f-4605-9fa2-ec32ffe1a94a)

**Comment by rithviknishad** [comment]
@AshrafMd-1 Let me rephrase how to replicate the issue:


## Method 1:

Create a "NORMAL" log update

This form does not collect IO balance related information from the user, so we are NOT supposed to say that the IO Balance was ZERO at this point since we are not aware if it was actually zero since data wasn't collected.

But still we see a record in the chart that it was 0, even though no information about it was collected.

## Method 2:

Create a "Critical Care" log update, but do not fill the I/O balance page and submit it.

Same situation here, information was not collected, so we are not supposed to represent that the IO balance was 0 during this log update. But still chart shows 0 in that record.
